pipeline/stop_places/sql/stop_places.sql

```diff
@@ -557,36 +557,42 @@ LANGUAGE plpgsql IMMUTABLE STRICT;
 
 
 /* 
- * Create a function, that estimates the duration based on the length of the path link.
- * The seconds are rounded up to the next integer.
+ * Create a function, that estimates the duration in seconds based on the length of the path link.
+ * Returns the seconds as integer (rounded up to the next integer).
+ * Special case elevator: the duration is estimated based on the number of levels that are passed.
  */
-CREATE OR REPLACE FUNCTION estimateDuration(geo geometry, walking_speed NUMERIC) RETURNS INTERVAL AS
+CREATE OR REPLACE FUNCTION estimate_duration(tags jsonb, geo geometry, level NUMERIC, walking_speed NUMERIC) RETURNS INTEGER AS
```

Comment:
Can we keep returning an `INTERVAL` here?

I guess you can then also omit or change some of the trailing casts in the function body.

Reply:
Added in [794b496](https://github.com/OPENER-next/osm2vdv462/commit/794b49679eb365ffa1f3e6b07df9c4b2785d92fd)